pycbc/events/single.py

```diff
@@ -232,10 +291,14 @@ def calculate_ifar(self, sngl_ranking, duration):
             )
             return None
 
-        rate_louder = rate * fits.cum_fit('exponential', [sngl_ranking],
-                                          coeff, thresh)[0]
+        rate_louder = rate * fits.cum_fit(
+            'exponential',
+            [sngl_ranking],
+            coeff,
+            thresh
+        )[0]
 
         # apply a trials factor of the number of duration bins
         rate_louder *= len(rates)
 
-        return conv.sec_to_year(1. / rate_louder)
+        return min(conv.sec_to_year(1. / rate_louder), self.maximum_ifar)
```

Comment:
This avoids stupid-loud IFAR for singles, especially when we can't get very high IFAR for backgrounds.